### PR TITLE
Chore: More robust identification of time/value fields in makeSeriesForLogs

### PR DESCRIPTION
**Pull request by kaydelaney**
**What this PR does / why we need it**:
Fixes an issue introduced by #23206 where `makeSeriesForLogs` was not updated to handle
new time/value order.

### CODE REVIEW

File: public/app/core/logs_model.ts [hunk 1]
```diff
@@ -136,30 +136,29 @@ export function makeSeriesForLogs(rows: LogRowModel[], intervalMs: number, timeZ
...
     // EEEP: converts GraphSeriesXY to DataFrame and back again!
     const data = toDataFrame(series);
-    const points = getFlotPairs({
-      xField: data.fields[1],
-      yField: data.fields[0],
-      nullValueMode: NullValueMode.Null,
-    });
 
-    const timeField = data.fields[1];
+    const timeField = data.fields.find(field => field.type === FieldType.time);
```
Comment (aocenas) [inline public/app/core/logs_model.ts:144]: Nitpick: We could use FieldCache.getFirstFieldOfType here.

**Review by aocenas — APPROVED** [review]
No need to change anything in Elastic or Influx?

File: public/app/core/logs_model.ts (hunk 1 above)
Comment (kaydelaney) [reply public/app/core/logs_model.ts:144]: oh nice, didn't know about this

**Review by ryantxu — APPROVED** [review]
LGTM

**Pull request merged**